collection/VideoCollection.js

```diff
@@ -0,0 +1,37 @@
+var VideoCollection = Backbone.Collection.extend({
+  model: VideoModel,
+  // query: this.$('.search').val(),
+
+  url: "https://www.googleapis.com/youtube/v3/search?key=AIzaSyBZv7oOxB5qYLsabM8t35bn2frCIjmDtqc&part=snippet&type=video&q=" + query,
+
+  parse: function(query) {
```

Comment:
The naming here is deceiving.  It would be more appropriately data, or dataFromServer.